lib/debug/thread_client.rb

```diff
@@ -1048,13 +1048,8 @@ def wait_next_action_
           when :call
             result = frame_eval(eval_src)
           when :irb
-            require 'irb' # prelude's binding.irb doesn't have show_code option
-            begin
-              result = frame_eval('binding.irb(show_code: false)', binding_location: true)
-            ensure
-              # workaround: https://github.com/ruby/debug/issues/308
-              Reline.prompt_proc = nil if defined? Reline
-            end
+            require_relative "irb"
+            activate_irb_integration
```

Comment:
could you make more clear name like `irb_integration.rb`?

Reply:
Sure, updated 👍